psalm.xml

```diff
@@ -47,5 +47,11 @@
                 <referencedClass name="Doctrine\Common\Cache\XcacheCache"/>
             </errorLevel>
         </UndefinedClass>
+        <UndefinedMethod>
+            <errorLevel type="suppress">
+                <!-- See https://github.com/doctrine/orm/issues/8884 -->
+                <referencedMethod name="Doctrine\DBAL\Platforms\AbstractPlatform::getGuidExpression"/>
+            </errorLevel>
+        </UndefinedMethod>
```

Comment:
Can you do the same for PHPStan? I can see an automatic comment above from that tool

Reply:
@greg0ire Added.